DAPM/DAPM.ClientApi/AccessControl/ApiHttpClient.cs

```diff
@@ -1,45 +1,60 @@
-using RabbitMQLibrary.Messages.AccessControl.Requests;
-using RabbitMQLibrary.Messages.AccessControl.Responses;
+using RabbitMQLibrary.Models.AccessControl;
 
 namespace DAPM.ClientApi.AccessControl;
 
 public class ApiHttpClient(IApiHttpClientFactory httpClientFactory) : IApiHttpClient
 {
-    private const string AddUserPipelineRoute = "add-user-pipeline";
-    private const string AddUserResourceRoute = "add-user-resource";
-    private const string AddUserRepositoryRoute = "add-user-repository";
-    private const string GetUserPipelinesRoute = "get-pipelines-for-user";
-    private const string GetUserResourcesRoute = "get-resources-for-user";
-    private const string GetUserRepositoriesRoute = "get-repositories-for-user";
+    private const string PipelineRoute = "pipeline";
```

Comment:
I would probably move these strings into a separate file to make it less bloated

Reply:
Fixed